### DISCUSSION TITLE
Cannot pair with an URI - Sample Application basic flows are not working.

### DISCUSSION THREAD

**Discussion by SangameswaranRS** [WalletConnect/walletconnect-monorepo#789] [Developer Support]
Validator.validateWCURI is throwing null, causing an exception while "pairing" with a URI.

URI from "example.walletconnet.com"

wc:1a1b83b4-226b-4bc8-b5d8-d18b313a9cb7@1?bridge=https%3A%2F%2Fv.bridge.walletconnect.org&key=414a1205200041715511be1d2368decd8c266d512b5cd2d749376dbbe70fd592

Is the kotlin SDK ready or WIP?

**Comment by SangameswaranRS** [comment]
Oh my bad, the example.walletconnet.com is v1 IG, is there a v2 version deployed?

**Reply by chadyj** [reply]
relay.walletconnect.com is for v2.

Be sure to check the docs https://docs.walletconnect.com/2.0/

The Kotlin v2 SDK is in beta.
